nc2zarr/cli.py

```diff
@@ -138,3 +140,112 @@ def nc2zarr(input_paths: Tuple[str],
 
 if __name__ == '__main__':
     nc2zarr()
+
+
+@click.command(name='nc2zarr-batch')
+@click.argument('config_template_path', metavar='CONFIG_TEMPLATE_PATH')
+@click.argument('config_path_template', metavar='CONFIG_PATH_TEMPLATE')
+@click.option('--range', '-R', 'ranges',
+              metavar='KEY MIN MAX', nargs=3, multiple=True,
+              help=f'Key value range assignments. MIN and MAX must be integers. Multiple may be given.')
```

Comment:
```suggestion
              help=f'Key value range assignments. MIN and MAX must be integers. Option may be repeated.')
```

I found the phrasing "Multiple may be given." hard to understand.